test/test_transforms.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (C) 2017 Christopher M. Biwer
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+import numpy
+import unittest
+from pycbc import conversions
+from pycbc import transforms
+from utils import parse_args_cpu_only
+from utils import simple_exit
+
+# list of transforms without an inverse function and to ignore
+NO_INVERSE = ["distance_to_redshift", "chi_p_to_cartesian_spin"]
+IGNORE = NO_INVERSE + ["aligned_mass_spin_to_cartesian_spin",
+                       "precession_mass_spin_to_cartesian_spin"]
```

Comment:
For `NO_INVERSE`, do a loop over the transforms and check if it has an inverse. Otherwise, whenever an inverse gets added, or when a new transform that has no inverse is added, people will probably forget to update this script. Also, why are the last two transforms added to `IGNORE`?

Reply:
Added loop. The other two were included because I didn't have time before. But they needed to have the primary/secondary parameters mapped correctly. Ie. they're given ``mass1`` and ``mass2`` which are not guaranteed to have ``mass1>mass2``. I've added codeblocks to handle this.